### ISSUE TITLE
Dark theme has some styling gaps

### ISSUE THREAD

**Issue by taksan** [stanfieldr/ghetto-skype#67]
The dark theme has a couple of messages that are not covered:

1) The join link in the group management menu is inside a panel that is still white and the text becomes invisible

2) If you click to share the conversation, it shows a "Your conversation is all set up" box, whose bluish color doesn't fit the dark theme.

I figured a way to fix these issues making the following css changes:

```
:root .swx .inviteLink
	background-color $background !important
	border-color $background !important

.welcomeMessage
	background-color #3d3d3d !important
 
```

**Comment by andrii-zymohliad** [comment]
I also have dark text on dark background in file accept dialog.

**Comment by Bukk94** [comment]
Hello Taksan, sorry for offtopic, but could you take a look on this issue of your API please?
https://github.com/taksan/skype-java-api/issues/81

**Comment by stanfieldr** [comment]
@andrii-zymohliad That's fixed in v1.4, but I'm releasing v1.4.1 in a day or two so you might want to hold off on upgrading.

@taksan Added your improvements and some others

**Comment by stanfieldr** [comment]
Included in [v1.4.1](https://github.com/stanfieldr/ghetto-skype/releases/tag/v1.4.1)
